scipy/stats/_entropy.py

```diff
@@ -117,6 +115,11 @@ def differential_entropy(
     axis : int, optional
         The axis along which the differential entropy is calculated.
         Default is 0.
+    method : str in {'vasicek', 'van es', 'ebrahimi', 'correa', 'auto'},
+    optional
```

Comment:
Actually do you know how to properly continue this line? It is showing up like:

![image](https://user-images.githubusercontent.com/6570539/115475786-46b10d80-a1f5-11eb-8592-7c54ad44e8f0.png)

Reply:
If we decide what to do here, I can remove the "We use high numbered seeds in order to have good entropy in the RNG" below at the same time.

Reply:
A work around would be to drop the "str in" since the set items already imply `str` type. I noticed that is what is done in many docstrings of scipy functions. Maybe even drop the space in "van es". Maybe this way optional might fit in the previous line.

Reply:
Thanks! That did it.

![image](https://user-images.githubusercontent.com/6570539/115502165-54808600-a229-11eb-93a9-d569b6c43d36.png)

Reply:
Ah sorry I missed this! There is a way, you just need to skip a blank line before the description. It's what I had to do for the `seed`:

```python
    seed : {None, int, `numpy.random.Generator`,
            `numpy.random.RandomState`}, optional

        If `seed` is None the `numpy.random.Generator` singleton is used.
        If `seed` is an int, a new ``Generator`` instance is used,
        seeded with `seed`.
        If `seed` is already a ``Generator`` or ``RandomState`` instance then
        that instance is used.
```

Reply:
Yeah I found that later tonight, too. But it's fine I think.

Reply:
Sure, sure 👍